install.sh

```diff
@@ -88,15 +98,15 @@ if [ -n "$HOSTNAME_PARAM" ] ; then
 	echo "-h $HOSTNAME_PARAM" >> "$HOST$DATADIR"/docker-run-opts
 	echo "$HOSTNAME_PARAM" > "$HOST$DATADIR"/hostname
 	OPTS="-h $HOSTNAME_PARAM"
-elif [ -z "$NET_HOST_PARAM" ] ; then
+elif [[ "$NET_PARAM" != "host" ]] ; then
 	echo "Please specify the hostname for the server with --hostname parameter." >&2
```

Comment:
Why not just `[ -z "$NET_PARAM" ]`? Why is `host` special?

Reply:
I don't have an in-depth knowledge of the docker run options but I think the container hostname will only be set if either `--hostname` or `--net host` is specified:

> With the network set to host a container will share the host’s network stack and all interfaces  from the host will be available to the container. The container’s hostname will match the hostname on the host system.

Reply:
Makes sense.